plain-pojos/src/main/resources/crypto.proto

```diff
@@ -1,9 +1,18 @@
 package plain.pojos;
 
+/**
```

Comment:
Why these? Works just fine without them

Reply:
It causes the fileds to be indexed, otherwise it works but does not use index.

Reply:
Is there a WARN message or something for such scenarios? I can see this being a very easy error to make by users.

Reply:
I have an ongoing PR for detecting such annotation missuse and there will be warnings in ISPN 9.3